media-gfx/gimp/gimp-2.10.14.ebuild

```diff
@@ -0,0 +1,230 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+GNOME2_EAUTORECONF=yes
+
+inherit eapi7-ver virtualx autotools gnome2 python-single-r1
+
+DESCRIPTION="GNU Image Manipulation Program"
+HOMEPAGE="https://www.gimp.org/"
+SRC_URI="mirror://gimp/v$(ver_cut 1-2)/${P}.tar.bz2"
```

Comment:
there is no reason to use `ver_cut` here, just make it `2.10`.

Reply:
Yes `doc` switch and its build deps are required to build develop-docs for 9999.ebuild. But for live-ebuild I shall create another PR as it contains new USE flags (new script language support), switching to gtk+3 etc. Anyway I must sync It with current where it possible.

Reply:
Done.